### ISSUE TITLE
Unknown license - Error message looks odd

### ISSUE THREAD

**Issue by jquorning** [alire-project/alire#1094]
When I enter an unknown license - Typolicense in this case - in a crate manifest and do ie:

`alr --no-tty index --check`

The error message looks odd:

```
ERROR: Loading release from /Users/jquorning/Alire/indexes/example/ex/example/example-0.0.0.toml:
ERROR:    licenses:
ERROR:    licenses:
ERROR:    Loading release from /Users/jquorning/Alire/indexes/example/ex/example/example-0.0.0.toml:
ERROR:    licenses:
ERROR:    licenses:
ERROR:    Invalid license expression 'Typolicense': Invalid license ID: 'Typolicense' (1:11) . SPDX expression expected (https://spdx.org/licenses/)
```

**Comment by Fabien-Chouteau** [comment]
Hi @jquorning,

Which part looks odd?

**Comment by jquorning** [comment]
The message has duplicate, and what look like debugging information.

I think this would be more appropriate:
```
ERROR: Loading release from /Users/jquorning/Alire/indexes/example/ex/example/example-0.0.0.toml:
ERROR:    Invalid license expression 'Typolicense': Invalid license ID: 'Typolicense' (1:11) . SPDX expression expected (https://spdx.org/licenses/)
```

**Comment by mosteo** [comment]
> The message has duplicate, and what look like debugging information.

This is a legitimate problem in how our scoping of errors works right now. It should be possible to remove these duplicates.

**Comment by mosteo** [comment]
Duplicate messages are fixed in #1490
